--- ./src/main-window.c.orig	2020-02-23 19:42:19.000000000 +0000
+++ ./src/main-window.c	2020-05-27 05:14:24.148951173 +0100
@@ -495,7 +495,7 @@ img_window_struct *img_create_window (vo
 	gtk_widget_show (separatortoolitem);
 	gtk_container_add (GTK_CONTAINER (img_struct->toolbar),separatortoolitem);
 
-	pixbuf = gtk_icon_theme_load_icon(icon_theme,"image", 20, 0, NULL);
+	pixbuf = gtk_icon_theme_load_icon(icon_theme,"image-x-generic", 20, 0, NULL);
 	tmp_image = gtk_image_new_from_pixbuf(pixbuf);
 	g_object_unref(pixbuf);
 
@@ -504,7 +504,7 @@ img_window_struct *img_create_window (vo
 	gtk_widget_set_tooltip_text(import_button, _("Import pictures"));
 	g_signal_connect ((gpointer)import_button, "clicked", G_CALLBACK (img_add_slides_thumbnails),img_struct);
 
-	pixbuf = gtk_icon_theme_load_icon(icon_theme,"sound", 20, 0, NULL);
+	pixbuf = gtk_icon_theme_load_icon(icon_theme,"audio-x-generic", 20, 0, NULL);
 	tmp_image = gtk_image_new_from_pixbuf(pixbuf);
 	g_object_unref(pixbuf);
 
